### ISSUE TITLE
Does the `bridge` service necessary at docker-compose.yml?

### ISSUE THREAD

**Issue by ArseniiPetrovich** [poanetwork/token-bridge#124]
As far as I see this service is exiting with non-failure code right after launch. It was required at the previous version of `docker-compose.yml` file and served as a launch point of `watchers` and `senders`. At the new version `watchers` and `senders` are considered as separate services, so I'm not sure if we still need the `bridge` service itself:
```
   build: .
    command: 'true'
    cpus: 0.1
    env_file: ./.env
    environment: [NODE_ENV=production]
    logging:
      driver: syslog
      options: {tag: '{{.Name}}/{{.ID}}'}
    mem_limit: 500m
    restart: on-failure
```

**Comment by akolotov** [comment]
@patitonar I think this question is really if new `docker-compose.xml` is still OK for the unit testing since initially the docker was integrated there and only after that was re-used to run the token bridge in the production.

**Comment by patitonar** [comment]
I think it's OK to remove the `bridge` service if it is not being used in the new version of `docker-compose.yml`. The e2e tests use another docker-compose file, defined on `e2e/docker-compose.yml` so it shouldn't affect the tests.

**Comment by ArseniiPetrovich** [comment]
Closed via #125
